sdk.js

```diff
@@ -0,0 +1,18 @@
+// const Twitter = require("twitter");
+
+// const client = new Twitter({
+//   consumer_key: process.env.API_KEY,
+//   consumer_secret: process.env.API_SECRET_KEY,
+//   access_token_key: process.env.ACCESS_TOKEN,
+//   access_token_secret: process.env.ACCESS_TOKEN_SECRET
+// });
+
+// const params = { screen_name: "pillarwallet" };
+
+// client.get("statuses/user_timeline", params, function(error, tweets, response) {
+//   if (!error) {
+//     console.log(tweets);
+//   }
+// });
+
+// module.exports = client;
```

Comment:
This whole file is commented out - it's generally good practice to remove _dead code_, as it's known.